microservices/password-manager/controllers/primistore/primistore-controller.ts

```diff
@@ -144,7 +147,7 @@ const encryptPasswordHandler = async (req, res, logger) => {
     .readFileSync(charsetPath)
     .toString("utf-8")
     .split("\n")
-    .slice(0, -1);
+    .slice(0, -1).join(" ");
   encryptedPassword = encryptWithCharset(charset, encryptedPassword.value);
```

Comment:
@frankhart2018 can you please have a look at this code, this seems to have incorrect type, what is the expected type goes here?

Reply:
also at line 132

Reply:
132 is a string, 146 is an array of strings